src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.commands;
+
+import duke.Duke;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exceptions.DukeException;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+public class AddCommand extends Command {
+    private Duke.TaskType taskType;
```

Comment:
If the `TaskType` class is to be used outside, should it still be placed inside `Duke`?
Additonally, is there a reason to couple `TaskType` with `Duke`?